Makefile

```diff
@@ -14,7 +14,6 @@ lint: ## run the fast go linters
 .PHONY: lint
 
 deps:
-	go get -u ./...
```

Comment:
should we have an upgrade-deps or something?

Reply:
yeah, though I need to think about it some more. I still have hope that dependabot can at some point work well for go projects.